Messages.py

```diff
@@ -264,7 +264,12 @@
     (0x9002, "\x08You are a \x05\x43FOOL\x05\x40!"),
     (0x9003, "\x08You found a piece of the \x05\x41Triforce\x05\x40!"),
     (0x9019, "\x08\x13\x09You found a \x05\x41Bombchu Bag\x05\x40!\x01It has some \x05\x41Bombchus\x05\x40 inside!\x01Find more in tall grass."),
-    (0x901A, "\x08You can't buy Bombchus without a\x01\x05\x41Bombchu Bag\x05\x40!")
+    (0x901A, "\x08You can't buy Bombchus without a\x01\x05\x41Bombchu Bag\x05\x40!"),
+    (0x908B, "\x08You got the\x01\x05\x41Ocarina A Button!\x05\x40\x01You can now play \x9F on the Ocarina!"),
```

Comment:
Message ID 0x908b is already used by chest game shuffle.

Reply:
Seeds should no longer generate if a duplicate message ID is used. That was part of the silver rupee PR.

Just added a new unit test for this #1977

Reply:
I already have this unit test on my branch. It did not catch this conflict because chest game shuffle patches messages using special code, not the messages lists.

Reply:
That should probably be changed in a future update. At very least have a placeholder in messages.py or something.